src/main/scala/firrtl/stage/FirrtlOptions.scala

```diff
@@ -0,0 +1,93 @@
+// See LICENSE for license details.
+
+package firrtl.stage
+
+import scala.util.Try
+
+import firrtl._
+import firrtl.options.{OptionsException, StageOptions}
+import firrtl.annotations.Annotation
+import firrtl.ir.Circuit
+import logger.LogLevel
+
+/** Internal options used to control the FIRRTL compiler stage
+  *
+  * @param topName the name of the top module
+  * @param globalLogLevel the verbosity of logging (default: [[logger.LogLevel.None]])
+  * @param classLogLevels the individual verbosity of logging for specific classes
+  * @param logToFile if true, log to a file
+  * @param logClassNames indicates logging verbosity on a class-by-class basis
+  * @param programArgs explicit program arguments
+  * @param inputFileNameOverride input FIRRTL file, default: `targetDir/topName.fir`
+  * @param outputFileNameOverride output file, default: `targetDir/topName.SUFFIX` with `SUFFIX`
+  *                               as determined by the compiler
+  * @param outputAnnotationFileName the output annotation filename, default: `targetDir/topName.anno`
+  * @param compilerName which compiler to use (default: "verilog")
+  * @param infoModeName the policy for generating [[firrtl.ir.Info]] when processing FIRRTL (default: "append")
+  * @param customTransforms any custom [[Transform]] to run
+  * @param firrtlSource explicit input FIRRTL as a [[scala.String]]
```

Comment:
I think this is Predef.String, or just String

Reply:
Neither seems to get unidoc to do the right thing. I'm leaving this off. I don't see this being used in the Scala documentation.